styles.css

```diff
@@ -5,7 +5,7 @@ body{
     max-width: 100%;
   }
   h1{
-    font-size: 40px;
+    font-size: 39px;
```

Comment:
I think you should be keen when writing codes to avoid blank lines that make your project not professional .